src/components/ViewAgents/MyWizards/index.tsx

```diff
@@ -129,26 +133,29 @@ function MyWizards() {
   }, [isError]);
 
   return (
-    <div className="my-wizards__wrapper">
-      {renderBody()}
-      <div className="my-wizards__footer">
+    <div className="">
+      <div>
         <div>
-          <div className="my-wizards__footer__title">Create an Ai Agent</div>
-          <div className="my-wizards__footer__description">
-            Craft and fine tune your agent
-          </div>
+          <h2>Agents</h2>
```

Comment:
move to `translation.json`

Reply:
@pratheeshme can you do this along with the svgs in your ui changes? Currently i have only placed placeholder text